.eslintrc.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
       }
     ],
     "capitalized-comments": "off",
-    "class-methods-use-this": "error",
+    "class-methods-use-this": "off",
```

Comment:
I am generally against linting rules that that go beyond formatting and performance -- and this one in particular seems to be purely a matter of judgement. We also disable it in so many places, that it's easier to just leave it off, and make sure methods are written properly when reviewing PRs IMO 😄 
Anyway, I'll probably move this change to a separate PR, since we also want to implement the rule about single quotes.